data/types/blood/glucose/glucose.go

```diff
@@ -33,3 +36,75 @@ func (g *Glucose) Normalize(normalizer data.Normalizer) {
 		g.Value = dataBloodGlucose.NormalizeValueForUnits(g.Value, units)
 	}
 }
+
+func roundToEvenWithDecimalPlaces(v float64, decimals int) float64 {
+	if decimals < 1 {
+		return math.RoundToEven(v)
+	}
+	coef := math.Pow(10, float64(decimals))
+	return math.RoundToEven(v*coef) / coef
+}
+
+type Classification string
+
+const (
+	ClassificationInvalid Classification = "invalid"
+
+	VeryLow       = "very low"
+	Low           = "low"
+	OnTarget      = "on target"
+	High          = "high"
+	VeryHigh      = "very high"
+	ExtremelyHigh = "extremely high"
+)
+
+type classificationThreshold struct {
+	Name      Classification
+	Value     float64
+	Inclusive bool
+}
+
+type Classifier []classificationThreshold
+
+func (c Classifier) Classify(g *Glucose) (Classification, error) {
+	normalized, err := dataBloodGlucose.NormalizeValueForUnitsSafer(g.Value, g.Units)
+	if err != nil {
+		return ClassificationInvalid, errors.Wrap(err, "unable to classify")
+	}
+	// Rounded values are used for all classifications. To not do so risks introducing
+	// inconsistency between frontend, backend, and/or other reports. See BACK-3800 for
+	// details.
+	rounded := roundToEvenWithDecimalPlaces(normalized, 1)
+	for _, threshold := range c {
+		if threshold.Includes(rounded) {
+			return threshold.Name, nil
+		}
+	}
+	// Ensure your highest threshold has a value like math.MaxFloat64 to avoid this.
+	return ClassificationInvalid, errors.Newf("unable to classify value: %v", *g)
```

Comment:
To remove this edge case (and remove the `math.MaxFloat64`) can we just remove the last `classificationThreshold` below and assume that if it isn't yet classified then in must be `ExtremelyHigh`?